### ISSUE TITLE
little question :)

### ISSUE THREAD

**Issue by shaharbukra** [nickg30/instaBot#1]
hi,
look like a nice idea :)

you just put it on your favorite and run it?

**Comment by nickg30** [comment]
yeah, browse a hashtag you like, click the most recent image, open the console, paste the code and enter!
